### DISCUSSION TITLE
onClick is deprecated, please use onPress

### DISCUSSION THREAD

**Discussion by bllakcn** [heroui-inc/heroui#3193] [Feedback]
My first time using the `as` prop to render a `Link` component as a `Button`.
I see this in my console
`onClick is deprecated, please use onPress` from useLink.

Is there anything I did wrong on my side?
```html
<Button
  as={Link}
  color='primary'
  onPress={doSomething}
  href={/}
  >
  Go somewhere
</Button>
```

**Comment by wingkwong** [comment]
lets track in https://github.com/nextui-org/nextui/issues/4493
